### PR TITLE
C#: Handle tuple expressions

### PR DESCRIPTION
**Pull request by calumgrant**
Extract tuple expressions, for example the body of `default((int,string))`. Previously this expression would not be extracted and there would be an error in `csharp.log`.

Fixes https://github.com/github/codeql-csharp-team/issues/3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@
 
 # It's useful (though not required) to be able to unpack codeql in the ql checkout itself
 /codeql/
+.vscode/settings.json
+csharp/extractor/Semmle.Extraction.CSharp.Driver/Properties/launchSettings.json
```
Comment (hvitved) [inline .gitignore]: Just checking: Did you intend to add these lines?

Comment (hvitved) [inline .gitignore]: Just checking: Did you intend to add these lines?

Comment (calumgrant) [reply .gitignore]: Yes -  it didn't merit its own PR.

Comment (calumgrant) [reply .gitignore]: Yes

**Pull request merged**